PluralKit.Bot/Commands/SystemEdit.cs

```diff
@@ -249,12 +268,12 @@ public async Task SystemProxy(Context ctx)
             }
 
             var patch = new SystemGuildPatch {ProxyEnabled = newValue};
-            await _db.Execute(conn => _repo.UpsertSystemGuild(conn, ctx.System.Id, ctx.Guild.Id, patch));
+            await _db.Execute(conn => _repo.UpsertSystemGuild(conn, ctx.System.Id, guildId, patch));
 
             if (newValue)
-                await ctx.Reply($"Message proxying in this server ({ctx.Guild.Name.EscapeMarkdown()}) is now **enabled** for your system.");
+                await ctx.Reply($"Message proxying in this server ({guild.Name.EscapeMarkdown()}) is now **enabled** for your system.");
```

Comment:
can this say "in the server" instead of "in this server" if the command was ran in a DM?